### DISCUSSION TITLE
Chrome Driver issue(Mac) - sbase install chromedriver 110' returned non-zero exit status 1.

### DISCUSSION THREAD

**Discussion by abdurraheemj** [seleniumbase/SeleniumBase#1741] [Q&A] [answered]
Hi Michael,
I have been using SeleniumBase for more than 1 year, and I had no problem since I started using it. Recently having issues with the Chrome Driver. I have been looking for solutions and decided to temporarily disable the chrome driver update in my Azure Pipeline. Now, My tests are started failing due to this Driver issue. I need your help to solve this
<img width="1109" alt="Screen Shot 2023-02-08 at 6 27 22 PM" src="https://user-images.githubusercontent.com/69648204/217685396-bb040ee6-d1ba-4d53-a153-764d27ef0cdf.png">

FAILED test_homepage.py::homepage_test::test_1 - subprocess.CalledProcessError: Command 'sbase install chromedriver 110' returned non-zero exit status 1.

**Comment by mdmintz** [comment, marked as answer]
Hello, the Chromium Team recently changed how they packed drivers.
Upgrading to the latest version of SeleniumBase should fix your issue.

More info here: https://github.com/seleniumbase/SeleniumBase/issues/1663

**Comment by abdurraheemj** [comment]
Thanks Michael, problem solved

On Wed, Feb 8, 2023 at 6:49 PM Michael Mintz ***@***.***>
wrote:

> Hello, the Chromium Team recently changed how they packed drivers.
> Upgrading to the latest version of SeleniumBase should fix your issue.
>
> More info here: #1663
> <https://github.com/seleniumbase/SeleniumBase/issues/1663>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/seleniumbase/SeleniumBase/discussions/1741#discussioncomment-4912826>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQTL6TGIMV23FMUFR4YGPETWWQ5JDANCNFSM6AAAAAAUV5NMXQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
*Thanks & Regards,*

Raheem Jameel
Software QA Engineer
Apartment Home Living - CoStar Group

https://www.apartmenthomeliving.com

Contact : (510)586-2375
